app/src/main/res/values/strings.xml

```diff
@@ -366,22 +366,22 @@
     <string name="keyboard_theme_title">Keyboard theme</string>
     <string name="keyboard_keys_category">Keys</string>
     <string name="keyboard_auto_go_action_title">Auto key action</string>
-    <string name="keyboard_auto_go_action_summary">Action of the "Go" key performed automatically after pressing a "Field" key</string>
+    <string name="keyboard_auto_go_action_summary">\"Go\" key action after pressing a \"Field\" key</string>
     <string name="keyboard_key_vibrate_title">Vibratory keypresses</string>
     <string name="keyboard_key_sound_title">Audible keypresses</string>
     <string name="allow_no_password_title">Allow no master key</string>
-    <string name="allow_no_password_summary">Enable the \"Open\" button if no credentials are selected</string>
+    <string name="allow_no_password_summary">Shows \"Open\" button if no credentials are selected</string>
     <string name="delete_entered_password_title">Delete password</string>
     <string name="delete_entered_password_summary">Deletes the password entered after a connection attempt</string>
     <string name="enable_read_only_title">Write-protected</string>
     <string name="enable_read_only_summary">Open your database read-only by default</string>
     <string name="enable_auto_save_database_title">Autosave database</string>
```

Comment:
What tense is this in? What happens. Needs a comment.

Reply:
If this setting is checked, "Open" button is enabled when there is no password string in the editText.
If this setting is unchecked, "Open" button is disabled when there is no password string in the editText.
This is especially for devices that have a fingerprint scanner near the button and that activated it by mistake.

Reply:
@J-Jamet I didn't understand that, but avoiding "Enable" is better. :)

Reply:
"Allow to click \"Open\" button  if no credentials are selected" ?